forest/Cargo.toml

```diff
@@ -82,7 +85,7 @@ rand              = "0.8"
 tempfile          = "3"
 
 [features]
-default          = ["forest_fil_cns"]
+default          = ["forest_fil_cns", "forest_statediff"]
```

Comment:
`forest_statediff` should probably be an non-optional dependency.

Reply:
Thanks, I see the root cause now; that was my bandaid solution because I completely missed the optional flag in the dependencies.